.gitignore

```diff
@@ -1,6 +1,6 @@
 __pycache__
 .vscode
-.venv
+venv
```

Comment:
Just checking that you intended to remove the "."  If this is correct, then LGTM.

Reply:
intentional change.